api/rest/restcore/projects_rest.php

```diff
@@ -275,11 +275,17 @@ function rest_project_version_get( \Slim\Http\Request $p_request, \Slim\Http\Res
 	}
 
 	$t_version_id = $p_args['version_id'] ?? $p_request->getParam( 'version_id' );
+	$t_released = (bool)$_request->getQueryParam( 'released', true );
```

Comment:
This is wrong. As documented in version_get_all_rows() PHPDoc, *released* is ternary, see 

https://github.com/mantisbt/mantisbt/blob/65cf0d5bc2560f789a218e9e282604a3171d97df/core/version_api.php#L553-L557

so you can't just cast it to bool, because this excludes VERSION_ALL (null).

And I think the default should be `null`, not `true`, matching current API behavior.